contrib/cirrus/lib.sh

```diff
@@ -244,6 +245,28 @@ use_netavark() {
     # N/B: The CNI packages are still installed and available. This is
     # on purpose, since CI needs to verify the selection mechanisms are
     # functional when both are available.
+
+    # See ./contrib/cirrus/CIModes.md.
+    # Vars defined by cirrus-ci
+    # shellcheck disable=SC2154
+    if [[ "$CIRRUS_CHANGE_TITLE" =~ CI:[AN]V[AN]V= ]]; then
+        # shellcheck disable=SC2154
+        if [[ "$CIRRUS_PR_DRAFT" != "true" ]]; then
+            die "Magic 'CI:NVAV=*' string can only be used on DRAFT PRs"
+        fi
+
+        magickind=$(sed -r -e 's~(.*CI:[AN]V[AN]V=)(\w+)(.*)~\2~' <<<"$CIRRUS_CHANGE_TITLE")
```

Comment:
@edsantiago I thought it important to extract the kind of magic being requested so helpful error messages could be displayed.  I'm not sure if I'm extracting it the best way, though this seems to work :confused:

Reply:
[Example, catching 'foobar' magic.](https://cirrus-ci.com/task/5238151308902400)

Reply:
Nice.